web/src/components/navbar/NavMenuItem.tsx

```diff
@@ -22,15 +21,15 @@ export const NavMenuItem = ({
   key,
   dataTestid,
 }: navMenuItemProps): ReactElement => {
-  const highlitBackgroundClassWithBoldText = "bg-primary-extra-light text-bold ";
+  const highlitBackgroundClassWithBoldText = "bg-primary-extra-light-not-important text-bold ";
```

Comment:
[Dust] This isn't really wrong, it just feels verbose and clunky.
```suggestion
  const lightBackgroundAndBoldText = "bg-primary-extra-light-not-important text-bold ";
```